datanode/gateway/graphql/schema.graphql

```diff
@@ -1717,7 +1718,7 @@ type Market {
   ): [Trade!] @deprecated(reason: "Use tradesConnection for cursor based pagination instead")
 
   tradesConnection(
-    "DateRange to retrieve trades from/to"
+    "Date Range to retrieve orders from/to. Start and end time should be expressed as an integer value of nano-seconds passed the Unix epoch"
```

Comment:
```suggestion
    "Date range to retrieve orders from/to. Start and end time should be expressed as an integer value of nano-seconds past the Unix epoch"
```

this one is called trades, but has switched to being described as orders - should it go back to being trades?

Reply:
Yeah, bad copy and paste on my part, will fix it